### PR TITLE
Updating Harbor Governance

### PR DESCRIPTION
**Pull request by michmike**

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -3,31 +3,44 @@
...
+| ----------------------------- | :---------------------: |
+| Technical Lead | Daniel Jiang (reasonerjt) |
+| Chief Architects | Daniel Jiang (reasonerjt), Steven Zou (steven-zou) |
+| Harbor Core | Daniel Jiang (reasonerjt) |
+| Replication Working Group | Steven Zou (steven-zou), Wenkai Yin (ywk253100) |
+| Webhooks | MingMing Pei (mmpei), Yan Wang (wy65701436) |
+| Interrogation Service/Pluggable Scanners | Daniel Pacak (danielpacak), Zach Hill (zhill), Steven Zou (steven-zou) |
+| Setup, Installation, Upgrade Experience | Daojun Zhang (stonezdj) , Qian Deng (ninjadq) |
+| Helm Chart & HA | Wenkai Yin (ywk253100), Daniel Jiang (reasonerjt) |
+| Policy Engine | Nathan Lowe (nlowe), Yan Wang (wy65701436)  |
```
Comment (steven-zou) [inline MAINTAINERS.md]: Please append @steven-zou for the policy-engine component.

File: MAINTAINERS.md [hunk 2]
```diff
@@ -3,31 +3,44 @@
...
-
 ## Maintainers
 
 | Maintainer | GitHub ID | Affiliation |
-| ---------- | --------- | ----------- |
-| Daojun Zhang | [stonezdj](https://github.com/stonezdj) | [VMware](https://www.github.com/vmware/) |
-| Wenkai Yin | [ywk253100](https://github.com/ywk253100) | [VMware](https://www.github.com/vmware/) |
-| Yan | [wy65701436](https://github.com/wy65701436) | [VMware](https://www.github.com/vmware/) |
+| --------------- | --------- | ----------- |
+| De Chen | [cd1989](https://github.com/cd1989) | Independent |
```
Comment (renmaosheng) [inline MAINTAINERS.md]: should we update De Chen's new profile?

**Comment by xaleeks** [comment]
@everyone, please don't merge until all maintainers have had chance to review and give feedback, we want all stakeholders accounted for.

File: MAINTAINERS.md (hunk 2 above)
Comment (michmike) [reply MAINTAINERS.md]: @renmaosheng can you give me the details please and i can update it...

File: MAINTAINERS.md [hunk 3]
```diff
@@ -3,31 +3,44 @@
...
+| Wenkai Yin | [ywk253100](https://github.com/ywk253100) | [VMware](https://www.github.com/vmware/) |
+| Henry Zhang | [hainingzhang](https://github.com/hainingzhang)| [VMware](https://www.github.com/vmware/) | 
+| Daojun Zhang | [stonezdj](https://github.com/stonezdj) | [VMware](https://www.github.com/vmware/) |
+| Steven Zou | [steven-zou](https://github.com/steven-zou) | [VMware](https://www.github.com/vmware/) |
 
 ## Emeritus Maintainers
 
 N/A
 
+| Harbor Technical Feature Area | Tech Lead |
```
Comment (hainingzhang) [inline MAINTAINERS.md]: "Tech Lead" can be changed to "Lead(s)"

File: MAINTAINERS.md [hunk 4]
```diff
@@ -3,31 +3,44 @@
...
+| Setup, Installation, Upgrade Experience | Daojun Zhang (stonezdj) , Qian Deng (ninjadq) |
+| Helm Chart & HA | Wenkai Yin (ywk253100), Daniel Jiang (reasonerjt) |
+| Policy Engine | Nathan Lowe (nlowe), Yan Wang (wy65701436)  |
+| Scheduling Engine | Steven Zou (steven-zou) |
+| Security | Daniel Pacak (danielpacak), Michael Michael (michmike) |
+| User Interface/Usability/User Experience | Meina Zhou (zhoumeina) |
+| Documentation | Stuart Clements (stuclem) |
+| Releases | Yan Wang (wy65701436), Steven Ren (renmaosheng) |
+| QA, Test Automation | Danfeng Liu (danfengliu) |
+| Community Manager | Jonas Rosland (jonasrosland) |
```
Comment (hainingzhang) [inline MAINTAINERS.md]: Please add "Henry Zhang" to Community Management:
Henry Zhang ( hainingzhang )

File: MAINTAINERS.md (hunk 2 above)
Comment (michmike) [reply MAINTAINERS.md]: @renmaosheng what's the new profile?

Comment (steven-zou) [reply MAINTAINERS.md]: Based on the current situation, De's case is `Independent` maintainer as his new company `Tencent` does not assign a maintenance mission to him.

Comment (michmike) [reply MAINTAINERS.md]: got it. i will leave it as is then.

**Comment by michmike** [comment]
Voting results:
VMware --> yes
Aqua --> Yes
Hyland --> Yes

@mmpei , @kofj , @cd1989 please review as well

**Comment by mmpei** [comment]
lgtm

**Comment by michmike** [comment]
100% agreement from Harbor maintainers. merging the PR now.
thank you all!

**Pull request merged**